iotdb-core/datanode/src/main/java/org/apache/iotdb/db/queryengine/plan/execution/config/executor/ClusterConfigTaskExecutor.java

```diff
@@ -1655,6 +1660,91 @@ public SettableFuture<ConfigTaskResult> createPipe(CreatePipeStatement createPip
     return future;
   }
 
+  @Override
+  public SettableFuture<ConfigTaskResult> alterPipe(AlterPipeStatement alterPipeStatement) {
+    SettableFuture<ConfigTaskResult> future = SettableFuture.create();
+
+    // Get pipe static meta
+    final String pipeName = alterPipeStatement.getPipeName();
+    final PipeStaticMeta pipeStaticMeta = PipeAgent.task().getPipeStaticMeta(pipeName);
```

Comment:
The pipeTaskAgent may not know this... Like when previous synchronizations are failed due to a dead address at connector: Consider this, a "createPipe" successfully handshakes with the receiver before configNode knows it, but the receiver is crushed just before the "CreatePipeProcedureV2" push the pipeMeta to dataNode. In this circumstance, the pipeTaskAgent does not know this pipe. You should allow users to alter a pipe like this that can not be created, and consequently won't be recorded at pipeTaskAgent.